logic/suggestions/utils.go

```diff
@@ -1,22 +1,22 @@
-package strategy
+package suggestions
 
 import (
 	"errors"
 
 	"github.com/joshprzybyszewski/cribbage/model"
 )
 
-func otherOptions(desired int, avoid map[model.Card]struct{}) [][]model.Card {
+func otherOptions(desired int, exclude map[model.Card]struct{}) [][]model.Card {
```

Comment:
This function feels like a more specific relative of `chooseNFrom`. Would it be advantageous/possible to use `exclude` here to build a set of cards to search through an pass `desired` and this set along to `chooseNFrom`?

Reply:
@joshprzybyszewski rip this comment :(